specification/src/test/groovy/io/datanerds/rappelkiste/specification/GetCounterSpecification.groovy

```
@@ -0,0 +1,66 @@
+package io.datanerds.rappelkiste.specification
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.jayway.restassured.RestAssured
+import com.jayway.restassured.response.Response
+import spock.lang.Narrative
+import spock.lang.Title
+
+import static io.datanerds.rappelkiste.specification.utils.JsonMatcher.aValidJsonString
+import static org.awaitility.Awaitility.await
+import static org.hamcrest.MatcherAssert.assertThat
+import static org.hamcrest.Matchers.equalTo
+import static org.hamcrest.Matchers.is
+
+
+@Narrative("Testing the Get part of the Counter Service")
+@Title("Get Counter Testsuite")
+class GetCounterSpecification extends BaseSpecification {
+
+    def "Retrieving an existing Counter"(String baseUrl) {
+
+        given: "A preexisting counter, with a uuid"
+        URI postUri = new URI(baseUrl + counterPath)
+        def JsonNode uuid = objectMapper.readTree(RestAssured.post(postUri).body.asString())
+        URI getUri = new URI(baseUrl + counterPath + "/" + uuid.asText())
+
+        await().ignoreExceptions().until({
```

We don't need to ignore exceptions

```
await().until({
            RestAssured
                    .expect()
                        .statusCode(200)
                    .when()
                        .get(getUri)
                    .andReturn()
        })
```